pages/common/recsel.md

```diff
@@ -0,0 +1,16 @@
+# recsel
+
+> Print records from a human-editable, plain text database called a recfile.
+> More information <https://www.gnu.org/software/recutils/manual/recutils.html>.
+
+- Extract name and version field from the output of guix:
+
+`guix package -s malloc | recsel -p name,version`
```

Comment:
What can be changed here? We should wrap it in the `{{token_syntax}}` so that tldr clients know and can highlight it as such.

Same goes for all the other examples.

Also, generally we don't showcase the usage of more than 1 command on a page. e.g. on this page we would _only_ have examples of the `recsel` command. If possible, it would be great to refactor this page a bit there, but if not it should be fine. This seems like an exception if 1 command can't be used without the other :-)

Reply:
So I've removed all references for `guix` since you can definitely use `recsel` without `guix`.